_pages/cps-iot-2024/agenda.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: page
+title:  
+description: Tock CPS-IoT Tutorial Program Agenda - May 13 2024
+permalink: /cps-iot-2024/agenda
+---
+
+# Agenda
+The RUSTIC tutorial will take place on Monday, May 13, 2024 at CPS-IoT Week. The schedule for the event can 
+be found below. Further information on the event can be found [here](https://tockos.org/cps-iot-2024).
+
+## Location
+
+To be announced.
+
+## Schedule
+
+<style type="text/css" scoped>
```

Comment:
scoped is a deprecated attribute and may not be supported.

In general we should avoid all this inline styling, both in the inline style tag, and the table.